### PR TITLE
C13806: Inconsistent punctuation mark on target versions due to aster…

### PR DESCRIPTION
**Pull request by carolinacmoravia**
…isks which need to be escaped

Hello, @mairaw,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by rpetrusha** [comment]
Thanks, @carolinacmoravia, for identifying this issue. I'm not quite sure what the fix is, though, since I'm not certain about the actual path to the log files. @merriemcgaw, do you happen to know? (This content evidently hasn't been updated since 4.5.)  The text is:
`%temp%\Microsoft .NET Framework 4.5*.txt`
`%temp%\Microsoft .NET Framework 4.5*.html`
I'm not sure what \* signifies here:
- Is there a missing \\ character (%temp%\\Microsoft .NET Framework 4.5\\\*.txt, for example), so that the log files are placed in a subdirectory named Microsoft .NET Framework 4.5?
- Is \* a wildcard for a part of the filename that's appended to "Microsoft .NET Framework 4.5" (e.g., "Microsoft .NET Framework 4.5 download.txt")?
- Where are log files currently placed and what form do they have. Should this be changed to "Microsoft .NET Framework \<version>*.txt"?

**Comment by merriemcgaw** [comment]
@rpetrusha  the asterisk is representing a wildcard in the filename.  I think you should replace 4.5 with <version> as well, since it has the version in the generated file name. The customer can also specify a log file to any location and name they choose.

**Comment by rpetrusha** [comment]
Thanks, @merriemcgaw. I've revised the section, if you'd like to take a look. @carolinacmoravia, I've made some additional changes to your PR. Also, since this is a new repo, can you sign the license agreement?

File: docs/framework/deployment/guide-for-administrators.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ This step-by-step article describes how a system administrator can deploy the [!
   
 8.  Complete the wizard.  
   
- The packagenow contains all the information you need to silently deploy the .NET Framework 4.5. Before you deploy the package and program, verify that it was installed on the distribution point; see the "Monitor Content" section of [Operations and Maintenance for Content Management in Configuration Manager](http://technet.microsoft.com/library/gg712694.aspx#BKMK_MonitorContent) in the Configuration Manager Documentation Library.  
+ The package now contains all the information you need to silently deploy the .NET Framework 4.5. Before you deploy the package and program, verify that it was installed on the distribution point; see the "Monitor Content" section of [Operations and Maintenance for Content Management in Configuration Manager](http://technet.microsoft.com/library/gg712694.aspx#BKMK_MonitorContent) in the Configuration Manager Documentation Library.
```
Comment (merriemcgaw) [inline docs/framework/deployment/guide-for-administrators.md]: I see that we are specifically referring to .NET 4.5 in this paragraph and in several others. Do we want to fix that up to be more generic across the whole article, or are we just focusing on the Log Files section?

**Comment by merriemcgaw** [comment]
@rpetrusha I saw a bunch of different references for 4.5 specifically throughout the document. Should we update the article in general to make it more generic.

**Comment by carolinacmoravia** [comment]
Hello, @rpetrusha and @merriemcgaw, 
Many thanks for your explanations and support on this. Could you confirm when the file is fixed, please?
Thanks again.

**Comment by rpetrusha** [comment]
Closing and reopening to see if CLA will appear as signed.

**Comment by rpetrusha** [comment]
At the moment, @merriemcgaw, I've just been focusing on the log section. This topic as a whole needs revision; we have an issue open for it and it's on my to do list, so I'll revise it then.

**Comment by merriemcgaw** [comment]
@rpetrusha Sounds good to me.

**Comment by rpetrusha** [comment]
@carolinacmoravia, could you sign the Contributor License Agreement, please? Just open the **Some checks haven't completed yet** section and click on the **Details** link for **license/cla** to sign. As soon as it's signed, I'll merge the PR.

File: docs/framework/deployment/guide-for-administrators.md [hunk 2]
```diff
@@ -225,12 +225,16 @@ This step-by-step article describes how a system administrator can deploy the [!
...
- %temp%\Microsoft .NET Framework 4.5*.html  
+ %temp%\Microsoft .NET Framework *version*\*.txt  
+ %temp%\Microsoft .NET Framework *version*\*.html  
   
- You can use the [log collection tool](http://www.microsoft.com/download/details.aspx?id=12493) to collect the [!INCLUDE[net_v45](../../../includes/net-v45-md.md)] log files and to create a compressed cabinet (.cab) file that reduces the size of the files.  
+ where *version* is the version of the .NET Framework that you're installing, such as 4.5 or 4.7.2.  
+ 
+ You can also specify the directory to which log files are written by using the `/log` command-line option in the .NET Framework installation command. For more information, see [.NET Framework deployment guide for developers](deployment-guide-for-developers.md#command-line-options). 
+ 
+ You can use the [log collection tool](http://www.microsoft.com/download/details.aspx?id=12493) to collect the .NET Framework log files and to create a compressed cabinet (.cab) file that reduces the size of the files.
```
Comment (pkulikov) [inline docs/framework/deployment/guide-for-administrators.md]: nit: the link can be *https*:
https://www.microsoft.com/download/details.aspx?id=12493

**Comment by rpetrusha** [comment]
Thanks, @carolinacmoravia! I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**